x-pack/plugins/search_playground/server/lib/get_chat_params.ts

```diff
@@ -82,6 +85,24 @@ export const getChatParams = async (
         type: 'anthropic',
       });
       break;
+    case GEMINI_CONNECTOR_ID:
+      llmType = 'gemini';
+      chatModel = new ActionsClientSimpleChatModel({
```

Comment:
I'd recommend updating from `ActionsClientLlm` to `ActionsClientSimpleChatModel` for Bedrock too, we got much better results [after this update](https://github.com/elastic/kibana/pull/182041)